lab_1_classify_by_unigrams/main.py

```diff
@@ -19,6 +24,17 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
+    if not isinstance(tokens, list) or not all(isinstance(el, str) for el in tokens):
+        return None
+    freqs = {}
+    for token in tokens:
+        if token in freqs:
```

Comment:
let's set 0 if key is not in the dict and increase the value always?

Reply:
done